mmdet/models/dense_heads/transformer_head.py

```diff
@@ -245,25 +248,339 @@ def forward_single(self, x, img_metas):
         # outs_dec: [nb_dec, bs, num_query, embed_dim]
         outs_dec, _ = self.transformer(x, masks, self.query_embedding.weight,
                                        pos_embed)
+
         all_cls_scores = self.fc_cls(outs_dec)
         all_bbox_preds = self.fc_reg(self.activate(
             self.reg_ffn(outs_dec))).sigmoid()
         return all_cls_scores, all_bbox_preds
 
-    @force_fp32(apply_to=('all_cls_scores', 'all_bbox_preds'))
+    @force_fp32(apply_to=('all_cls_scores_list', 'all_bbox_preds_list'))
+    def loss(self,
+             all_cls_scores_list,
+             all_bbox_preds_list,
+             gt_bboxes_list,
+             gt_labels_list,
+             img_metas,
+             gt_bboxes_ignore=None):
+        """"Loss function.
+
+        Defaultly only outputs from the last feature level is used for
```

Comment:
Only outputs from the last feature level are used for computing losses by default.